src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-service-broker-integration.mdx

```diff
@@ -0,0 +1,423 @@
+
+The broker exposes the New Relic service on the Marketplace and allows users to directly create a service instance and bind it to their apps either from Apps Manager or from the command line.
+
+The New Relic Service Broker for VMware Tanzu tile installs the New Relic Service Broker as an app, registers it as a service broker on VMware Tanzu, and exposes its service plans on the Marketplace. Each service plan is associated with an existing New Relic account, which is configured during the tile setup.
+
+Selecting a plan binds your app with the New Relic agent, and the agent starts reporting to the New Relic account which is associated with the selected plan. This makes the installation and subsequent use of New Relic on your VMware Tanzu apps easier and more straightforward.
+
+## Prerequisites [#prereqs]
+
+New Relic Nozzle for VMware Tanzu has the following requirements:
+
+* Service broker v1.12.18 and later
+* For service brokers on v1.12.12 and earlier, the unique GUIDs for plans were calculated differently. For these plans to not break compatibility, the GUIDs must be the same as before. The migration script preserves the plan GUIDs for existing plans in the plan collection for v1.12.12 and earlier.
+
+<Collapser
+    id="broker-legacy"
+    title="Obtain key for service brokers v1.12.12  and earlier"
+>
+You need to override your plan GUID if you've changed the original license key that was associated with a service broker on v1.12.12 or earlier. To obtain the original plan GUID from Cloud Controller, run the following script:
+
+```shell
+   cf curl $(cf curl /v2/services?q=label:newrelic | grep “service_plans_url” |
+
+    awk ‘{print $2}’ | sed ‘s/[",]//g’) | egrep “"name":|"unique_id":” |
+
+    sed ‘s/[",]//g’ | tr -s " " | awk ’ {name=$0; getline; printf(\t%-40s %-40s \n”,name,$0)}’
+   ```
+
+There are two new properties labeled pre-1.12.12 plan? and Plan Guid Override (broker 1.12.12 or older) in the plan collection for each plan in the tile configuration. Do not change either of these properties because they are set internally where required.
+
+* In New Relic Service Broker v1.12.13 and later, leave pre-1.12.12 plan unchecked, and Plan Guid Override blank.
+* The tile is supported on Ops Manager v2.9.x, v2.10.x, and v3.0.x.
+* You can install the tile on any of these versions, and upgrade from v1.9.x to any Ops Manager version up to and including v3.x.
+* No upgrade paths are required for older versions of the tile, since versions older than v1.9.0 are not supported.
+* v1.12.6 and later of the tile support migration from older versions of the tile, and preserve existing services and service plans.
+* If you are using tiles older than v1.11.4, you must first upgrade to v1.11.4, then to v1.12.9, then to the latest version of the tile.
+</Collapser>
+
+
+The following table provides version and version-support information about the service broker for VMware Tanzu.
```

Comment:
Do users need this information to get started? Are these the prereqs needed so this tech can collect data?

Reply:
Yes.